fe/fe-core/src/main/java/org/apache/doris/analysis/ShowColumnStatsStmt.java

```diff
@@ -36,6 +36,8 @@
 import org.apache.doris.statistics.AnalysisManager;
 import org.apache.doris.statistics.ColStatsMeta;
 import org.apache.doris.statistics.ColumnStatistic;
+import org.apache.doris.statistics.PartitionStatistic;
```

Comment:
PartitionStatistic and PartitionStatisticCacheKey are column level, not table level. How about renaming to xxxColumnStatistic